### DISCUSSION TITLE
Album metadata display adjustments in Artist view

### DISCUSSION THREAD

**Discussion by rocketpear** [ciderapp/Cider#739] [Feature Request]
In the artist view under album arts, the displayed metadata contains album artist, which is redundant information (the user is already browsing that specific artist's page). This information could be replaced with album year, which is arguably more important information and helps the user quickly glance at a specific artist's history of releases.
This way of displaying metadata is used in the Spotify Windows app and I believe that in AM Mac app as well.

**Current view:**
![image](https://user-images.githubusercontent.com/22647650/160648628-b1fdaf70-a3f3-414d-a1c0-d5e21a1b0b8d.png)

**Spotify solution:**
![image](https://user-images.githubusercontent.com/22647650/160648795-ca2a2507-4002-4f43-b5a3-cda440721107.png)

**Comment by KillerParticles** [comment]
I do agree it makes more sense. If this also was to change for Artists view however I think there should be a link to their artist page.
